utils/location.ts

```diff
@@ -1,8 +1,17 @@
 export const isLocation = () => {
-    const location = window.location.pathname;
-    if(location.indexOf('folder') > -1) {
-        return true;
-    } else {
-        return false;
-    }
-}
\ No newline at end of file
+  const location = window.location.pathname;
+  if (location.indexOf("folder") > -1) {
+    return true;
+  } else {
+    return false;
+  }
+};
+
+export const isblock = () => {
+  const location = window.location.pathname;
+  if (location.indexOf("sign") > -1) {
+    return false;
+  } else {
+    return true;
+  }
```

Comment:
문자열에 사용 가능한 includes 매서드를 사용하면 바로 boolean을 반환하므로 if를 사용한 분기처리가 필요없다는 장점이 있습니다 🤗

```suggestion
 location.includes("folder")
```